DOCS-441 update Storage-provisioning.md (#104)
aerospike/kubernetes-operator: docs/Storage-provisioning.md

## Local volume

This example uses a local SSD (identified as `/dev/nvme0n1`). Attach this SSD to each Kubernetes worker node which will be used for getting the primary storage device for the Aerospike cluster deployment.

### Create a Discovery Directory

Before deploying local volume provisioner, create a discovery directory on each worker node and link the block devices to be used in the discovery directory. The provisioner will discover local block volumes from this directory.

```shell
$ lsblk
NAME    MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
nvme0n1       8:16   0  375G  0 disk
nvme0n2       8:32   0  375G  0 disk
```

```shell
mkdir /mnt/disks
sudo ln -s /dev/nvme0n1 /mnt/disks/
sudo ln -s /dev/nvme0n2 /mnt/disks/
```

:::note

> You can use also your own discovery directory, but make sure that the provisioner is also configured to point to the same directory.

Reviewer: ```suggestion
You can also use your own discovery directory, but make sure that the provisioner is configured to point to the same directory.
```

:::

### Deploy the Local Volume Provisioner

To automate the local volume provisioning, we will create and run a provisioner based on [kubernetes-sigs/sig-storage-local-static-provisioner](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner).

> The provisioner will run as a DaemonSet which will manage the local SSDs on each node based on a discovery directory, create/delete the PersistentVolumes, and clean up the storage when it is released.

Reviewer: ```suggestion
The provisioner runs as a DaemonSet which manages the local SSDs on each node based on a discovery directory, creates and deletes the PersistentVolumes, and cleans up the storage when it is released.
```

The local volume static provisioner for this example is defined in [aerospike_local_volume_provisioner.yaml](https://github.com/aerospike/aerospike-kubernetes-operator/tree/2.0.0-rc1/config/samples/storage/aerospike_local_volume_provisioner.yaml).

The storage class is defined in [local_storage_class.yaml](https://github.com/aerospike/aerospike-kubernetes-operator/blob/master/config/samples/storage/local_storage_class.yaml). This and other example CRs are stored in [the main Aerospike Kubernetes Operator repository](https://github.com/aerospike/aerospike-kubernetes-operator/tree/master/config/samples).

Create local storage class, then deploy the provisioner.

```shell
kubectl create -f local_storage_class.yaml

kubectl create -f aerospike_local_volume_provisioner.yaml
```

> Verify the persistent volumes have been created.

Reviewer: ```suggestion
Verify the persistent volumes were created.
```
